### ISSUE TITLE
Timezone support

### ISSUE THREAD

**Issue by romiem** [yaru22/angular-timeago#80]
Is it possible to have timeago detect the browsers timezone and adjust the display text accordingly?

**Comment by astik** [comment]
Timeago works comparing the JS date object with a new date object (now), so that, timezone have no impact on the result. If i am not wrong, a date instance can be seen as a timestamp, no depending of the timezone. Timezones only affect a date when it is displayed, so tat it won't affect the difference of two date.
Can you provide us a detail use case with what are the inputs and what are the expectations ?

**Comment by romiem** [comment]
You are correct- the issue I had was related to the fact that I was trying I compare a date object with a time zone against a date without a time zone (and hence it was using the users' OS time zone). 

TL;DR- this is not an issue with angular-timezone
